pandas/core/ops.py

```diff
@@ -293,6 +309,18 @@ def _get_frame_op_default_axis(name):
 Returns
 -------
 result : DataFrame
+
+Examples
+--------
+>>> a = pd.Series([1, 1, 1, np.nan], index=['a', 'b', 'c', 'd'])
```

Comment:
Can you use a DataFrame for this example? It can just be a single-column DataFrame with these same values and index.